### PR TITLE
Return empty link if no foreign object

### PR DESCRIPTION
**Pull request by sebheitzmann**
please see #469

### CODE REVIEW

**Comment by sergeyz** [comment]
Hi, @sebheitzmann, please check failed build https://travis-ci.org/orocrm/platform/builds/138038136

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for some time for the submission to be processed.

**Comment by artml** [comment]
@sergeyz are we going to merge this?

File: src/Oro/Bundle/DataGridBundle/Extension/Formatter/Property/LinkProperty.php [hunk 1]
```diff
@@ -35,6 +35,11 @@ public function getRawValue(ResultRecordInterface $record)
 
         try {
             $label = $record->getValue($this->getOr(self::DATA_NAME_KEY) ?: $this->get(self::NAME_KEY));
+            
+            // if the foreign object doesn't exist there is no link possible
+            if (!$label) {
+                return "";
```
Comment (sergeyz) [inline src/Oro/Bundle/DataGridBundle/Extension/Formatter/Property/LinkProperty.php]: We use N\A over system if value is not provided

**Comment by sergeyz** [comment]
@artml yes, lets do it.

**Comment by sergeyz** [comment]
Merged to dev repository, thanks!

**Pull request closed without merging**